helper.h

```diff
@@ -175,4 +175,25 @@ class Helper{
                 refresh_screen(60);
             }
         }
+
+        void play_intro()
```

Comment:
Definitely need to comment this, from my understanding you are covering the splash-screen with a white rectangle and fading it out.

Reply:
Cheers Riley, Yes that's right, I have commented the function as suggested.